app/views/hyrax/collections/show.html.erb

```diff
@@ -55,6 +55,22 @@
 	<div class="row hyc-body">
 		<div class="col-md-8 hyc-description">
 			<%= render 'collection_description', presenter: @presenter %>
+
+			<% if @presenter.collection_type_is_nestable? && @parent_collection_count > 0 %>
+					<div class="row hyc-blacklight hyc-bl-title">
```

Comment:
You might be able to remove the `row` class here, and remove the `<div class="col-md-12"></div>` below, to keep the markup cleaner.

Reply:
done